aptos-move/framework/aptos-framework/sources/stake.spec.move

```diff
@@ -190,6 +190,7 @@ spec aptos_framework::stake {
     }
 
     spec on_new_epoch {
+        pragma verify = false;
```

Comment:
I am unable to identify how the on_new_epoch function calls coin::mint in order to ensure the correct aborts_if condition.

Reply:
Is it possible to add requires instead of aborts_if? If it does not work, try to change the spec of mint instead of modifying the spec of the caller

Reply:
Resolved by adding the appropriate aborts_if clauses.